### PR TITLE
freebayes-0.9.21-goolf-1.4.10.eb (WIP)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 0 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/8af7ddb9be7c7d411cd4 for a full test report.

File: easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+version = '0.9.21'
+
+homepage = 'https://github.com/ekg/freebayes'
+description = """ Bayesian haplotype-based polymorphism discovery and genotyping. """
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'} 
+
+# no direct download url. As described in documentation the download has been done with
+# $> git clone --recursive git://github.com/ekg/freebayes.git
+# Then compressed and placed in the sources folder with name freebayes-0.9.21.tar.bz2
```
Comment (boegel) [inline easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb]: use https://github.com/ekg/freebayes/archive/v0.9.21.tar.gz ?

File: easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+sources = [SOURCELOWER_TAR_BZ2]
+
+builddependencies = [('CMake', '2.8.12', '', ('GCC', '4.8.4'))]
```
Comment (boegel) [inline easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb]: bump CMake version?

File: easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+parallel = 1
+
+# workaround to avoid the error "The C compiler identification is unknown"
```
Comment (boegel) [inline easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb]: this is a bit weird... defining `$CC` as `gcc` makes the build fail?!

File: easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb (hunk 1 above)
Comment (pescobar) [inline easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb]: that's not enough. See that the required command line is "git clone --recursive" to also download git submodules not included in the release tarball

File: easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb (hunk 3 above)
Comment (pescobar) [inline easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb]: to be honest I don't remember why I had to do this. It was a quick workaround so the build works. I need to review it with more detail....

File: easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+prebuildopts = "unset CC && unset CXX && "
+
+files_to_copy = ["bin", "README.md", "examples", "scripts", "python"]
+
+sanity_check_paths = {
+        'files': ["bin/freebayes", "bin/bamleftalign"],
+        'dirs': [""]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb]: 4-space indent, and investigate the $CC? this requires more investigation

minor changes needed here, put this in WIP plz!

**Comment by boegel** [comment]
@pescobar: ping on remarks?

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**